tests/by-util/test_csplit.rs

```diff
@@ -1359,3 +1359,11 @@ fn precision_format() {
         assert_eq!(at.read("xx 0x001"), generate(10, 51));
     }
 }
+
+#[test]
+fn zero_error() {
+    let (_, mut ucmd) = at_and_ucmd!();
+    ucmd.args(&["in", "0"])
+        .fails()
+        .stderr_contains("0: line number must be greater");
+}
```

Comment:
This test looks incomplete/incorrect because GNU `csplit` fails with a different error:
```
$ csplit in 0
csplit: cannot open 'in' for reading: No such file or directory
```
The file has to exist to get the expected error message:
```
$ touch in
$ csplit in 0
csplit: 0: line number must be greater than zero
```

Reply:
oh, fun :)
well spotted!

Reply:
I fixed it too and added a test for this